textproc/grc/Portfile

```diff
@@ -70,5 +59,14 @@ post-destroot {
         INSTALL \
         README.markdown \
         Regexp.txt \
+        TODO \
         ${destroot}${docdir}
 }
+
```

Comment:
the notes are redundant now, correct?

Reply:
as i wrote in my previous comment, I am under the impression it is not redundant.
> ps, i just left the note I added since it is from the upstream and made me think a MacPorts user using python27 must be aware they could be some weird/wrong behaviours using py27 (not to mention it's likely py27 support is deemed to be dropped sooner than later. please let me know if that makes sense and it's ok for you.

nevertheless, I rely on your final judgement, after all, you looks just a wee savvier than me ;-)

Reply:
I'd say we don't care about Python 2 compatibility since we don't run it under Python 2.7; you just changed the port to use Python 3.9. I don't see a reason to keep the notes.

Reply:
Yah, I think @reneeotten is right here.

Reply:
done! thank you very much to both of you for the help and support!